src/bundle/Resources/public/js/scripts/fieldType/ezcountry.js

```diff
@@ -0,0 +1,45 @@
+(function (global) {
+    const SELECTOR_FIELD = '.ez-field-edit--ezcountry';
+
+    class EzCountryValidator extends global.eZ.BaseFieldValidator {
+        /**
+         * Validates the select box value
+         *
+         * @method validateSelection
+         * @param {Event} event
+         * @returns {Object}
+         * @memberof EzCountryValidator
+         */
+        validateSelection(event) {
+            const isRequired = event.target.required;
+            const isEmpty = !event.target.value.trim().length;
+            const isError = (isEmpty && isRequired);
+            const label = event.target.closest(SELECTOR_FIELD).querySelector('.ez-field-edit__label').innerHTML;
+            const result = { isError };
+
+            if (isEmpty) {
```

Comment:
I think it should be `isError`

Reply:
I disagree. I'll keep that like it is now. By checking if a field value is empty I'm able to attach a correct error message.

Reply:
If we want to be precise with adding correct error message then we should use `isEmpty && isError` because when the field is not required the value can be empty.